research-hub-web/src/app/components/articles/articles.component.html

```diff
@@ -142,7 +142,8 @@ <h4>You might also be interested in<hr/></h4>
                 *ngFor="let relatedItem of article.relatedItemsCollection.items" 
                 class="collection">
                 
-                    <a [routerLink]="'/' + relatedItem['__typename'].toLowerCase() + '/' + relatedItem['slug']">
+                    <!-- <a [routerLink]="'/' + relatedItem['__typename'].toLowerCase() + '/' + relatedItem['slug']"> -->
+                    <a [routerLink]="['/', relatedItem['__typename'].toLowerCase(), relatedItem['slug']]">
```

Comment:
Personal preference but seems tidier to me :shrug: Thoughts? Couldn't update the `equipment.component.html`s one as the page's broken at the moment.

Reply:
Yeah looks good, not a critical change anyway